spec/latest/json-ld-api/index.html

```diff
@@ -876,7 +877,7 @@
         <li>If <em>value</em> contains the key <code>@type</code>:
           <ol class="algorithm">
             <li>Initialize <em>type</em> to the value associated with the
-              <code>@type</code> key, which must be a <a>string</a>. Otherwise, an
+              <code>@type</code> key, which MUST be a <a>string</a>. Otherwise, an
```

Comment:
This is a property of a conformant document, not a requirement of the algorithm. So shouldn't this remain a lowercase must?
(same elsewhere)